src/main.ts

```diff
@@ -95,21 +96,28 @@ async function bootstrap() {
   const expressSessionSecret = configService.get<string>(
     "expressSessionSecret",
   );
+  const mongoUrl = configService.get<string>("mongodbUri");
+
   if (expressSessionSecret) {
     app.use(
       session({
         secret: expressSessionSecret,
         resave: false,
         saveUninitialized: true,
+        store: MongoStore.create({
+          mongoUrl, // MongoDB connection string
+          collectionName: "sessions", // Collection name for storing sessions
+          ttl: 24 * 60 * 60, // Session TTL (24 hours)
+        }),
+        cookie: {
+          secure: true,
+        },
```

Comment:
According to this post, it is possible to have only one connection to the database.
https://stackoverflow.com/questions/66731669/mongoose-and-connect-mongo

Can you look into it and see if we can reduce duplication?